.github/workflows/ansible-test.yml

```diff
@@ -220,7 +233,10 @@ jobs:
 
       # Run the integration tests
       - name: Run integration test
-        run: ansible-test integration -v --color --retry-on-error --continue-on-error --diff --python ${{ matrix.python }} --docker --coverage
+        env:
+          # set to any container alias usable by ansible-test --docker
+          ANSIBLE_TEST_CONTAINER: ${{ matrix.container }}
+        run: ansible-test integration -v --color --retry-on-error --continue-on-error --diff --python ${{ matrix.python }} --docker $ANSIBLE_TEST_CONTAINER --coverage
```

Comment:
@felixfontein it was sort of an artifact of how I got there, not initially using the matrix. Anyway it will probably be moot when https://github.com/ansible/default-test-container/pull/60 lands, so I'll likely end up reverting all this anyway